python/lsst/ap/association/diaCalculationPlugins.py

```diff
@@ -358,12 +362,16 @@ def calculate(self,
             if pTileName not in diaObjects.columns:
                 diaObjects[pTileName] = np.nan
 
-        def _fluxPercentiles(df):
-            pTiles = np.nanpercentile(df["psFlux"], self.config.percentiles)
-            return pd.Series(dict((tileName, pTile)
-                                  for tileName, pTile in zip(pTileNames, pTiles)))
+        with np.warnings.catch_warnings():
+            np.warnings.filterwarnings('ignore', r'All-NaN slice encountered')
```

Comment:
Can we please use double quotes to be consistent with the rest of the file?  Also, why is this a raw string?

Reply:
Done. The raw string is based on what I saw in examples both online and in our own code: https://github.com/lsst/pipe_tasks/blob/448f008be835c65bd1e4254d7fddb65aacc18f68/python/lsst/pipe/tasks/functors.py#L541

Reply:
Odd. It can't be doing anything useful can it? Sounds like cargo cult 😄